src/middlewares/requestMiddlware.js

```diff
@@ -3,7 +3,8 @@ import sendResult from '../utils/sendResult';
 
 const requestMidd = {
   async checkExistingTrip(req, res, next) {
-    const { requestId } = req.params;
+    let { requestId } = req.params;
+    requestId = (parseInt(requestId, 10)) || -1;
```

Comment:
nice work! I need some clarification on this. Is that mean that when a requestId is not a number, we should replace it by -1. If that's the case then I think instead of doing that you should send an error message to the user.